cmd/common.go

```diff
@@ -6,9 +6,17 @@ import (
 	"os"
 	"path/filepath"
 
+	"github.com/rancher/types/apis/management.cattle.io/v3"
 	"github.com/urfave/cli"
 )
 
+var sshCliOptions = []cli.Flag{
```

Comment:
is the cli flag needed since you already add the option to cluster.yml

Reply:
It is a convenience thing. If I download the yaml, I can just run `rke up --ssh-agent-auth` instead of having to edit the file. If the work flow for RKE is simliar to other devops tools different team members might have different setups. You and I apparently would have to edit the file and each time we ran a common yaml as we found out with this issue.